src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialog.fxml

```diff
@@ -0,0 +1,111 @@
+                    </tooltip>
+                </Button>
+            </GridPane>
+            <StackPane VBox.vgrow="ALWAYS">
+                <Accordion fx:id="accordion" minHeight="300.0" prefHeight="300.0" maxHeight="300.0">
+                    <panes>
+                        <TitledPane fx:id="filePane" text="%Currently unlinked files">
+                            <ScrollPane fitToWidth="true">
+                                <CheckTreeView fx:id="unlinkedFilesList"/>
+                            </ScrollPane>
+                        </TitledPane>
+                        <TitledPane fx:id="resultPane" text="%Import result" disable="true">
+                            <TableView fx:id="importResultTable">
+                                <columns>
+                                    <TableColumn fx:id="colStatus" prefWidth="100.0" text="%Status"/>
+                                    <TableColumn fx:id="colMessage" prefWidth="300.0" text="%Message"/>
+                                    <TableColumn fx:id="colFile" prefWidth="500.0" text="%File"/>
+                                </columns>
+                                <columnResizePolicy>
+                                    <TableView fx:constant="CONSTRAINED_RESIZE_POLICY"/>
+                                </columnResizePolicy>
+                            </TableView>
+                        </TitledPane>
+                    </panes>
+                </Accordion>
+                <VBox fx:id="progressPane" alignment="CENTER" StackPane.alignment="CENTER">
+                    <ProgressIndicator fx:id="progressDisplay" maxHeight="50.0" maxWidth="50.0"
+                                       prefHeight="50.0" prefWidth="50.0" progress="0.0"/>
+                    <Label fx:id="progressText" text="%Searching file system..."/>
+                </VBox>
+            </StackPane>
+            <ButtonBar buttonMinWidth="80.0" BorderPane.alignment="CENTER">
+                <buttons>
+                    <Button onAction="#selectAll" text="%Select all" ButtonBar.buttonData="OTHER"/>
+                    <Button onAction="#unselectAll" text="%Unselect all" ButtonBar.buttonData="OTHER"/>
+                    <Button onAction="#collapseAll" text="%Collapse all" ButtonBar.buttonData="OTHER"/>
+                    <Button onAction="#expandAll" text="%Expand all" ButtonBar.buttonData="OTHER"/>
+                    <Button onAction="#exportSelected" text="%Export selected" fx:id="exportButton" ButtonBar.buttonData="OTHER">
+                        <tooltip>
+                            <Tooltip text="%Export to text file."/>
+                        </tooltip>
+                    </Button>
+                    <Button onAction="#startImport" text="%Import" fx:id="importButton" ButtonBar.buttonData="APPLY">
+                        <tooltip>
+                            <Tooltip text="Starts the import of BibTeX entries."/>
```

Comment:
```suggestion
                            <Tooltip text="%Starts the import of BibTeX entries."/>
```

Please undo the removal in `JabRef_en.properties`

Reply:
There are no bibtex entries here, it's an import of files, right?

Reply:
These are bibtex entries. Extracted from xmp or pdf or a bib file.